src/Xamarin.Android.Build.Tasks/Utilities/MarshalMethodsAssemblyRewriter.cs

```diff
@@ -95,10 +123,161 @@ void MoveFile (string source, string target)
 				Files.CopyIfChanged (source, target);
 				try {
 					File.Delete (source);
-				} catch (Exception ex) {
+				} catch (Exception) {
```

Comment:
Instead of removing `ex`, we should add a `Log.LogDebugMessage (ex)` to the `catch` block.